themes/bootstrap3/templates/RecordDriver/LibGuides/result-list.phtml

```diff
@@ -10,5 +10,12 @@
         </a>
       </h2>
     </div>
+    <?php foreach ($this->driver->getSummary() as $summaryItem): ?>
+      <?php if (!empty($summaryItem)): ?>
+        <div class="resultItemLine1">
```

Comment:
I have seen this `resultItemLine1` class used elsewhere, even repeated in a for loop in an [EDS template](https://github.com/vufind-org/vufind/blob/dev/themes/bootstrap3/templates/RecordDriver/EDS/result-list.phtml#L32).  So just following an apparent pattern.  This could just as easily be a `<p>` with no class.

Reply:
I don't really like this class very much, and I think there was some discussion recently about possibly doing away with it... but if you want to follow the precedent, it shouldn't hurt anything.